js/src/dropdown.js

```diff
@@ -218,12 +218,20 @@ class Dropdown extends BaseComponent {
       return
     }
 
+    // If this is a touch-enabled device we remove the extra
+    // empty mouseover listeners we added for iOS support
+    if ('ontouchstart' in document.documentElement) {
+      [].concat(...document.body.children)
+        .forEach(elem => EventHandler.off(elem, 'mouseover', null, noop()))
+    }
+
```

Comment:
I think, till next pr this is not valid. you call it in the `hide` function, it is not triggered by any event ;)